docker-compose.test.yml

```
@@ -22,16 +30,21 @@
       - '9324:9324'
     environment:
       - NEXT_PUBLIC_PRODUCTION_API_URL=http://prod:3001
+    healthcheck:
+      <<: *default-healthcheck
+      test: ["CMD","curl","-f","http://localhost:9324"]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)